types/global.d.ts

```diff
@@ -0,0 +1,6 @@
+// Types for compiled templates
+declare module 'ember-modifier/templates/*' {
```

Comment:
This confuses me, can you explain what this does? Also, AFAIK this package doesn't have a `addon/templates` folder...

Reply:
Great catch; I'll remove it from this PR. In general, this is generated by e-c-ts so that folks don't get type errors for importing templates, e.g. to set `layout` on components in addons.